### PR TITLE
[spirv] Add support for -fvk-invert-y

### PR DESCRIPTION
**Pull request by antiagainst**
This is to accommodate Vulkan's coordinate system, which is different
from DX's.

Fixes https://github.com/Microsoft/DirectXShaderCompiler/issues/959

### CODE REVIEW

**Comment by antiagainst** [comment]
@marcelolr: this PR adds a new option: `-fvk-invert-y` to accommodate Vulkan's coordinate system, as explained in #959.

File: tools/clang/lib/SPIRV/SPIRVEmitter.cpp [hunk 1]
```diff
@@ -395,6 +395,9 @@ SPIRVEmitter::SPIRVEmitter(CompilerInstance &ci,
       seenPushConstantAt(), needsLegalization(false) {
   if (shaderModel.GetKind() == hlsl::ShaderModel::Kind::Invalid)
     emitError("unknown shader module: %0", {}) << shaderModel.GetName();
+  if (options.invertY &&
+      !(shaderModel.IsVS() || shaderModel.IsDS() || shaderModel.IsGS()))
```
Comment (ehsannas) [inline tools/clang/lib/SPIRV/SPIRVEmitter.cpp]: nit: Could simply be:
```cpp
if(options.invertY && !shaderModel.IsVS() && !shaderModel.IsDS() && !shaderModel.IsGS())
```

Comment (antiagainst) [reply tools/clang/lib/SPIRV/SPIRVEmitter.cpp]: Yep. Will fix in a following up PR.

Comment (antiagainst) [reply tools/clang/lib/SPIRV/SPIRVEmitter.cpp]: Done in #975

**Pull request merged**